src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx

```diff
@@ -159,46 +170,12 @@ APM agents offer detailed transaction trace visibility for individual services.
 
 ## How OpenTelemetry works with New Relic [#how-it-works]
 
-Here are two basic ways you could set up OpenTelemetry with New Relic: use the OpenTelemetry collector (recommended), or use the native OTLP endpoint (pre-release). Each approach assumes you set up instrumentation in your code, but they differ in how the data is gathered and sent to New Relic.
-
-If you are interested in tracing, we recommend that you instrument as many services as you can so that you get the most benefit from distributed tracing. There are two main options for trace sampling:
-
-* Configure the head-based, native sampling in OpenTelemetry, which means OpenTelemetry samples traces before they are sent to New Relic. Head-based sampling doesn’t analyze all traces, but instead randomly samples traces up front before details about the completed traces are known. Both the OpenTelemetry collector and the native OTLP endpoint support this option.
-* If you want New Relic to analyze all your traces, configure tail-based sampling with New Relic Infinite Tracing, which reroutes traces to our cloud-based trace observer. The trace observer accepts all your traces and sorts through them to find useful ones. If you want to know more about this option, especially if you want to use it in the EU, see [Introduction to Infinite Tracing](/docs/understand-dependencies/distributed-tracing/infinite-tracing/introduction-infinite-tracing). While Infinite Tracing is not yet compatible with the native OTLP endpoint, it is still possible to configure tail-based sampling via the collector, for more information see [Tail Sampling Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/tailsamplingprocessor).
-
-<Callout variant="tip">
-  We discuss some variations on these basic approaches in our [OpenTelemetry architecture recipes](/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-architecture-recipes). For a detailed discussion about the architecture of OpenTelemetry itself, see our [blog](https://newrelic.com/blog/best-practices/opentelemetry-opentracing-opencensus).
-</Callout>
+New Relic supports the native OpenTelemetry Protocol (OTLP) for for sending telemetry data. This allows you to use the vendor neutral components developed by the OpenTelemtry community to send your data to New Relic.
```

Comment:
repeat of the word "for" in the first sentence

Reply:
@kaileyhaynes, good catch! I fixed this!